bin/oref0-determine-basal.js

```diff
@@ -105,7 +106,7 @@ if (!module.parent) {
             console.error(msg.msg);
             // console.log(JSON.stringify(msg));
             if (!params['missing-meal-ok']) {
-              errors.push(msg);
+              warnings.push(msg);
```

Comment:
for interactive/manual mode strict is nice, since it will help spot issues, but when running a loop it's a problem

I added `--missing-meal-ok` to my rigs to prevent issues, but I expect others to miss that and have their loop fail.